src/Logic.hs

```diff
@@ -407,23 +419,44 @@ handleCommentAdded triggerConfig projectConfig prId author body state =
     -- a user has push access requires an API call.
     Just pr -> do
       let approvalType = parseMergeCommand triggerConfig body
-      isApproved <- if isJust approvalType
-        then isReviewer author
-        else pure False
+      isApproved <-
+        if isJust approvalType
+          then isReviewer author
+          else pure False
+      -- For now Friday at UTC+0 is good enough. 
+      -- See https://github.com/channable/hoff/pull/95 for caveats and improvement ideas.
+      day <- dayOfWeek . utctDay <$> getDateTime
       if isApproved
-        -- The PR has now been approved by the author of the comment.
-        then do
-          let (order, state') = Pr.newApprovalOrder state
-          state'' <- approvePullRequest prId (Approval author (fromJust approvalType) order) state'
-          -- Check whether the integration branch is valid, if not, mark the integration as invalid.
-          if Pr.baseBranch pr /= BaseBranch (Config.branch projectConfig)
-             then pure $ Pr.setIntegrationStatus prId IncorrectBaseBranch state''
-             else pure state''
+        then -- The PR has now been approved by the author of the comment.
+         case fromJust approvalType of 
+           -- To guard against accidental merges we make use of a merge window. 
+           -- Merging inside this window is discouraged but can be overruled with a special command.
+          (approval, OnFriday) | day == Friday -> handleMergeRequested projectConfig prId author state pr approval
+          (approval, NotFriday)| day /= Friday -> handleMergeRequested projectConfig prId author state pr approval
+          (other, NotFriday) -> do
+            () <- leaveComment prId ("Your merge request has been denied, because \
+                                      \merging on Fridays is not recommended. \
+                                      \To override this behaviour use the command `"
+                                      <> Pr.displayApproval other <> " on Friday`.")
+            pure state
+          (other, OnFriday) -> do
+            () <- leaveComment prId ("Your merge request has been denied because \
+                                      \it not Friday. Run " <> 
+                                      Pr.displayApproval other <> " instead")
```

Comment:
Good idea rejecting this as well so that people don't get conditioned to just always using the more powerful command :+1: